src/openklant/components/contactgegevens/api/viewset.py

```diff
@@ -19,40 +20,35 @@
 from openklant.components.token.permission import TokenPermissions
 
 
+@extend_schema(tags=["organisaties"])
+@extend_schema_view(
+    list=extend_schema(
+        summary="Alle organisaties opvragen.",
+        description="Alle organisaties opvragen.",
```

Comment:
I added the description so that the api spec is consistant with the legacy api specs. Because they also used the same description as summary if there wasn't additional information.